server/controllers/index.js

```diff
@@ -110,11 +108,13 @@ const controllers = {
 
   product: {
     get: (req, res) => {
-      const { itemID } = req.query;
+      console.log('recieved');
+      const itemID = req.query.itemID;
```

Comment:
Super minor, but since you can't control what a user will try to ping your endpoint with, do safe object value look ups, set sensible defaults

https://lodash.com/docs/4.17.15#get

Reply:
corrected in upcoming PR for production branch